pom.xml

```
@@ -72,30 +72,27 @@
     <project.majorVersion>6</project.majorVersion>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-    <pathling.sparkVersion>3.4.2</pathling.sparkVersion>
+    <pathling.sparkVersion>3.5.0</pathling.sparkVersion>
     <pathling.scalaVersion>2.12</pathling.scalaVersion>
     <pathling.scalaFullVersion>2.12.17</pathling.scalaFullVersion>
-    <pathling.deltaVersion>2.4.0</pathling.deltaVersion>
-    <!-- Hadoop is currently pinned to 3.3.4 because of this vulnerability: CVE-2022-26612 -->
+    <pathling.deltaVersion>3.1.0</pathling.deltaVersion>
```

```suggestion
    <pathling.deltaVersion>3.2.0</pathling.deltaVersion>
```

Can I sneak an update to the version released just a week ago in?

---

I've just made that change and it looks good!